.gitattributes

```diff
@@ -16,3 +16,6 @@ pkg/k8s/client/clientset/** linguist-generated
 pkg/k8s/client/informers/** linguist-generated
 pkg/k8s/client/listers/** linguist-generated
 *.bt linguist-language=D
+
+./install/kubernetes/cilium/values.yaml merge=ours
```

Comment:
Should this line be combined with the line at the top or should it be separate?

Is the `./` prefix vs. `/` prefix vs. no `./` meaningful?